src/biokbase/narrative/jobs/job.py

```diff
@@ -239,19 +240,10 @@ def __getattr__(self, name):
         return attr[name]()
 
     def __setattr__(self, name, value):
-        if name in STATE_ATTRS:
-            self._acc_state[name] = value
-        elif name in JOB_INPUT_ATTRS:
-            self._acc_state["job_input"] = self._acc_state.get("job_input", {})
-            self._acc_state["job_input"][name] = value
-        elif name in NARR_CELL_INFO_ATTRS:
-            self._acc_state["job_input"] = self._acc_state.get("job_input", {})
-            self._acc_state["job_input"]["narrative_cell_info"] = self._acc_state[
-                "job_input"
-            ].get("narrative_cell_info", {})
-            self._acc_state["job_input"]["narrative_cell_info"][name] = value
-        else:
-            object.__setattr__(self, name, value)
+        if name in ALL_ATTRS:
```

Comment:
force all job attribute updates to use the `update_state` method

Reply:
What's the difference in behavior here?